client/src/ui/NotFound.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function NotFound() {
+  return (
+    <div className="col-md-12 col-sm-12 col-xs-12 not-found-padding-top">
```

Comment:
```suggestion
    <div className="col-12 not-found-padding-top">
```

since we upgraded to Bootstrap 4 this semester, `col-xs-12` is now just `col-12` (see the [migration guide](https://getbootstrap.com/docs/4.0/migration/#grid-system) and the [Bootstrap 4 grid options documentation](https://getbootstrap.com/docs/4.0/layout/grid/#grid-options)). It also looks like you're applying the full column width to all screen sizes, so you can simplify it to just `col-12`.

Reply:
Fixed! Thanks